formatter/fmt

```diff
@@ -0,0 +1,58 @@
+#! /usr/bin/env bash
+
+set -e
+set +x
+set -o pipefail
+
+cd /code
+
+# Gets the list of file paths for files that are tracked by git and have
+# changes relative to the main branch.
+function get_tracked_modified_files() {
```

Comment:
Make this optional - when running in a PR, it should check all files

Reply:
why?

Reply:
For one, its faster than the other tests, so not a big deal to run it on every file.

For the other, If there's an a change made by the formatter in a PR test, then it will error.  So by checking all the files, we make sure that there's no errors that have snuck in. And if we add a new formatting option, we'd error out the test and know that our formatting was not correct with some file.

But I guess it's preference-  perhaps doing it this way and getting only updating the formatting of files when we change them for some other reason is OK too.

Reply:
ah i think we actually do test all of them since we don't use bin/fmt in CI but rather https://github.com/seattle-uat/civiform/blob/faster-autoformatter/.github/workflows/format.yaml#L23-L26